src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -990,12 +1011,6 @@ class="wp-interactivity-router-loading-bar"
 				data-wp-class--start-animation="state.navigation.hasStarted"
 				data-wp-class--finish-animation="state.navigation.hasFinished"
 			></div>
-			<div
```

Comment:
Sorry to bring this up after having already approved!

I've just noticed that we should rename the function to `print_router_loading_markup()`, update its docstring and update the reference where we pass this function to the filter [over here](https://github.com/WordPress/wordpress-develop/pull/7304/files#diff-ba219807876e59a40e6ccd29916a088f32a9951d2210209c15cba58c0a337657R1053).

Reply:
This is a public method, extenders could be calling it or modifying its behavior in filters. Renaming the method could be disruptive. It's true the method name isn't a perfect description now. What do you think of adding a note in the method documentation instead of changing the name?

Reply:
Maybe we could do this:
- Create a new `print_router_markup` function
- Deprecate the “old” method, and have it log a deprecation.
- The “old” method calls  the new `print_router_markup` function.

I think that might be the way to go.

I’m looking at some existing functions/methods that have the @deprecated docstring. For example this function is deprecated, calls `_deprecated_function()` and then calls the updated version:

https://github.com/WordPress/wordpress-develop/blob/9e290835d6f006997d59ebaf063df113bb15a363/src/wp-includes/class-wp-scripts.php#L479-L505

Reply:
Ok, I've searched for the method name and it only appears in Gutenberg across searchable [plugins](https://wpdirectory.net/search/01J8QN3A1M1XEH7HVE1E3G3109) and [themes](https://wpdirectory.net/search/01J8QN3B0VC884R6D4AACRQ5QC).

Let's just rename it.

Reply:
All right!